horace_core/horace_function_utils/private/is_horace_data_object.m

```diff
@@ -1,5 +1,5 @@
 function [ok,sqw_obj,dnd_obj]=is_horace_data_object(w)
```

Comment:
Calls to this function, or its implementation, can be replaced with this function collapses to `isa(w, 'SQWDnDBase')`

Reply:
true